build.sbt

```diff
@@ -358,7 +358,8 @@ lazy val asyncHttpClient = libraryProject("async-http-client")
       nettyBuffer,
       nettyCodecHttp,
       reactiveStreams,
-    )
+    ),
+    Test / parallelExecution := false
```

Comment:
Would be interesting to figure out why this must be, but that's for another time.